test/e2e/util.go

```diff
@@ -181,8 +183,8 @@ func defaultExternalDNS(name, zoneID, zoneDomain string) operatorv1alpha1.Extern
 	}
 }
 
-func routeExternalDNS(name, zoneID, zoneDomain string) operatorv1alpha1.ExternalDNS {
-	return operatorv1alpha1.ExternalDNS{
+func routeExternalDNS(name, zoneID, zoneDomain, routerName string) operatorv1alpha1.ExternalDNS {
+	resoure := operatorv1alpha1.ExternalDNS{
```

Comment:
`resoure`?! You wanted to say `resource` I think. Something like `extDNS` can be even more meaningful.

Reply:
this resource itself not needed, we will use direct return.
was mis-directed with previous test failure